packages/signer-web/src/createWebAuthClient.ts

```diff
@@ -1,4 +1,9 @@
-import { AuthClient, OauthCancelledError } from "@alchemy/signer";
+import {
+  AuthClient,
+  OauthCancelledError,
+  type CreateTekStamperFn,
+  type CreateWebAuthnStamperFn,
+} from "@alchemy/signer";
```

Comment:
heads up that this import will change to `@alchemy/auth` if #2063 lands first. otherwise I will update this in that pr